Documentation/TypoScript/Index.rst

```diff
@@ -0,0 +1,170 @@
+:navigation-title: TypoScript introduction
+..  include:: /Includes.rst.txt
+
+..  _typoscript:
+
+=========================================
+A minimal page created by pure TypoScript
+=========================================
+
+TypoScript is the basic configuration script used to configure the frontend
+output of a page in TYPO3.
+
+You can find detailed information about
+:ref:`TypoScript <t3coreapi:typoscript-syntax-syntax>` and its syntax in
+the reference "TYPO3 explained".
+
+The semantics of TypoScript settings are listed in the
+:ref:`TypoScript Reference <t3tsref:start>`.
+
+..  _typoscript-hello-world:
+
+"Hello world" example in TypoScript
+===================================
+
+..  versionchanged:: 13.1
+    The site set can be used as
+    :ref:`TypoScript provider <t3coreapi:site-sets-typoscript>`. It is not
+    necessary to use a TypoScript record as is explained in tutorials for
```

Comment:
"as is explained" sounds strange - maybe "as explained"?

Reply:
done